Core/Commands/CustomCommands.py

```diff
@@ -619,3 +619,21 @@ def subreddit(bot, event, *args):
                                                              hangups.SegmentType.LINK,
                                                              link_target=link_url)])
 
+@DispatcherSingleton.register
+def subverse(bot, event, *args):
```

Comment:
Oh right, I forgot about auto replies. I already modified Dispatcher.py to
respond to /v/, but using auto replies would be much more elegant, thanks
On Jul 15, 2015 4:23 PM, "Wardell Bagby" <notifications@github.com> wrote:

> In Core/Commands/CustomCommands.py
> <https://github.com/ShaunOfTheLive/HangoutsBot/pull/31#discussion_r34724393>
> :
>
> > @@ -619,3 +619,21 @@ def subreddit(bot, event, *args):
> >                                                               hangups.SegmentType.LINK,
> >                                                               link_target=link_url)])
> >
> > +@DispatcherSingleton.register
> > +def subverse(bot, event, *args):
>
> Perhaps you could also expand upon this command by adding in a autoreply
> that triggers from /v/*? So that way users could go /v/subverse and it
> would also link?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShaunOfTheLive/HangoutsBot/pull/31/files#r34724393>.
>